.env.example

```diff
@@ -5,21 +5,15 @@ APP_DEBUG=true
 VITE_DEBUG=true
 DEBUGBAR_ENABLED=false
 
-MAGENTO_HOST="magento.local"
-RAPIDEZ_HOST="rapidez.local"
+MAGENTO_HOST="magento.localhost"
+RAPIDEZ_HOST="rapidez.localhost"
```

Comment:
I think `.test` is a better option as probably 99% is using Laravel Herd or Laravel Valet